### DISCUSSION TITLE
i am getting https://rpc.walletconnect.com/v1/profile/.....?projectId=eaa7d52… 404 (Not Found)  in the console (multiple times).

### DISCUSSION THREAD

**Discussion by kr-arunp** [reown-com/appkit#2514] [Q&A]
- i am using wagmi and web3Modal
 - nextjs 14

**Comment by rtomas** [comment]
The functionality of your dApp is working ok ?

**Reply by kr-arunp** [reply]
@rtomas thanks.
The issue does not appear to affect the application's functionality. It seems to have been resolved.

Initially, the problem arose due to the behaviour of my application, which uses Next.js 14, wagmi. Previously, I was utilising a local storage-based wallet session. This setup led to numerous 404 errors during server-side rendering (SSR) redirects.

Currently, I have switched to using cookie-based wallet sessions. While this adjustment has significantly mitigated the issue, some residual 404 errors persist, though they are less frequent. Notably, these errors are mainly observed during the initial load in the browser's network tab. Additionally, the issue does not occur during SSR redirects or cache re-validations.
